src/locales/fr/translation.json

```diff
@@ -76,32 +93,42 @@
   "Required": "Obligatoire",
   "Research group": "group de recherche",
   "Results": "Resultats",
-  "This output of a mathematical model depends on model assumptions and parameter choices. We have done our best (in limited time) to check the model implementation is correct. Please carefully consider the parameters you choose and interpret the output with caution.": "Cette sortie d'un modèle mathématique dépend des hypothèses du modèle et des choix de paramètres. Nous avons fait de notre mieux (dans un temps limité) pour vérifier que la mise en œuvre du modèle est correcte. Veuillez considérer attentivement les paramètres que vous choisissez et interpréter la sortie avec prudence",
   "Run": "Lancer",
   "Scenario": "Scenario",
   "Seasonal forcing": "Forçage saisonnier",
   "Seasonal peak": "Pic saisonnier",
   "Select region for which to plot confirmed case and death counts.": "Sélectionnez la région pour laquelle tracer les cas confirmés de cas et de décès.",
   "Severe": "Sévère",
-  "Severity-assumptions": "Hypothèses de gravité",
+  "Severely ill": "Gravement malade",
+  "Severity assumptions and age-specific isolation": "Hypothèses de gravité et isolation par âge",
   "Severity of ICU overflow": "Gravité du débordement des soins intensifs",
+  "Show humanized results": "Montrer les résultats humanisés",
```

Comment:
This is difficult to translate it directly: here "humanisés" doesn't make much sense. I would suggest simply *Mettre en forme les résultats*